data/Application.css

```diff
@@ -40,3 +40,9 @@
 .content-area {
     padding: 1.33rem;
 }
+
+.copy-button {
+    background: none;
+    border: 1px solid @borders;
+    box-shadow: none;
+}
```

Comment:
I don't think we need to add this style because it actually makes the button much harder to read than our normal button style